plugins/toolbox/src/components/Generators/QR.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { DefaultEditor } from '../DefaultEditor';
+import { faker } from '@faker-js/faker';
+import QRCode from "react-qr-code";
+
+export const QR = () => {
+  const [input, setInput] = React.useState('');
+  const sample = faker.internet.url();
+
+
+  return (
+    <DefaultEditor
+      input={input}
+      setInput={setInput}
+      sample={sample}
+      rightContent={
+        <>
```

Comment:
Can be removed as not required.

Reply:
Please describe for me and continue review my code, many thanks